packages/metamask/yarn.lock

```diff
@@ -0,0 +1,2064 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We can go ahead and remove this from the repo, it will be built on publish of the package